src/Stratis.Bitcoin.Features.Miner/Staking/PosMinting.cs

```diff
@@ -1143,35 +1132,35 @@ public Models.GetStakingInfoModel GetGetStakingInfoModel()
         }
 
         /// <summary>
-        /// Checks whether the coinstake should be split or not. The rule is that we want to be splitting coins until they are individually around 100 Strats,
-        /// but aiming not to exceed a total amount of coins bigger than (500+1)*3. S
+        /// Checks whether the coinstake should be split or not.
         /// </summary>
         /// <param name="stakedUtxosCount">Number of non-empty UTXOs currently at stake.</param>
         /// <param name="amountStaked">Total amount currently at stake</param>
         /// <param name="coinValue">Value of the coin we are considering to split</param>
         /// <param name="splitFactor">number of time we want to split the coin</param>
         /// <param name="chainHeight">Current height of the chain</param>
         /// <returns><c>true</c> if the coinstake should be split, <c>false</c> otherwise.</returns>
-        /// <remarks>The coinstake is split if the number of non-empty UTXOs we have in the wallet is under the given threshold, or if amount is above a certain value.</remarks>
-        /// <seealso cref="CoinstakeSplitLimitMultiplier"/>
+        /// <remarks>
+        /// The rule is that we want to be splitting coins until they are individually around 100 Strats,
+        /// but aiming not to exceed a total amount of coins bigger than (<c><seealso cref="requiredCoinAgeForStaking"/></c>>+1)*<c><seealso cref="CoinstakeSplitLimitMultiplier"/></c>.
```

Comment:
also you don't have requiredCoinAgeForStaking as it is a local variable

So I would suggest something like
```
<remarks>
We do not split a coin if the value of new coins after the split would be less than <see cref="SplitCoinMinSize"/>. Because we split the coin to multiple outputs defined by split factor, we only consider coins with value at least <see cref="SplitCoinMinSize"/> * <see cref="SplitFactor"/>.
<para>
If the above-mentioned criteria is satisfied, then we split the coin if its value is greater than an expected average value of coins that we would have if we have perfect distribution of the value among all our coins while having a specific number of coins that we aim for. The optimal number of coins we are looking for is calculated based on consensus settings of coin maturity and minimum required coin age for staking.
</para>
</remarks>
<seealso cref="CoinstakeSplitLimitMultiplier"/>
<seealso cref="SplitFactor"/>
```